prov/verbs/src/fi_verbs.h

```diff
@@ -261,8 +261,10 @@ struct fi_ibv_eq {
 	struct fid_eq		eq_fid;
 	struct fi_ibv_fabric	*fab;
 	fastlock_t		lock;
+	ofi_atomic32_t		ref;
 	struct dlistfd_head	list_head;
 	struct rdma_event_channel *channel;
+	struct dlist_entry	domain_list;
```

Comment:
This list isn't really used.  The refcnt is sufficient.

Reply:
Thanks! That was a leftover from the last patch version.